hawc/apps/assessment/api.py

```diff
@@ -170,6 +175,60 @@ def public(self, request):
         serializer = serializers.AssessmentSerializer(queryset, many=True)
         return Response(serializer.data)
 
+    @method_decorator(cache_page(60 * 60 * 24))
+    @action(detail=False, methods=("get",), renderer_classes=PandasRenderers)
+    def training_data(self, request):
+        Endpoint = apps.get_model("animal", "Endpoint")
+        # map of django field names to friendlier column names
+        column_map = {
+            "assessment_id": "assessment_uuid",
+            "name": None,
+            "animal_group__experiment__chemical": "chemical",
+            "animal_group__species__name": "species",
+            "animal_group__strain__name": "strain",
+            "animal_group__sex": "sex",
+            "system": None,
+            "organ": None,
+            "effect": None,
+            "effect_subtype": None,
+            "animal_group__experiment__study__title": "study_title",
+            "animal_group__experiment__study__abstract": "study_abstract",
+            "data_location": None,
```

Comment:
let's rename `data_location` to `endpoint_data_location` to be clear what we're referring to